docs/getting-started/installation.md

```diff
@@ -11,11 +11,15 @@ the two sections below ("Fragmentation Backends" and "Reference Data
 Generators")
 
 ```shell
-mamba install -c conda-forge openff-bespokefit
+mamba install -c conda-forge openff-bespokefit "qcportal <0.50"
 ```
 
 If you do not have Mamba installed, see the [OpenFF installation documentation](openff.docs:install).
 
+:::{warning}
+
+Some upstream dependencies may not be supported on Apple Silicon. To force `mamba` to use the use of the Rosetta emulation layer, use `CONDA_SUBDIR=osx-64`, which is described in more detail [here](https://docs.openforcefield.org/en/latest/install.html#openff-on-apple-silicon-and-arm).
+
```

Comment:
I couldn't get the one-liners to work on `osx-arm64`. I think this is worth mentioning here whether or not everything works on Apple Silicon.